glfw/src/Window.zig

```diff
@@ -266,6 +271,120 @@ pub inline fn hint(h: Hint, value: anytype) Error!void {
+    accum_red_bits: c_int = 0,
+    accum_green_bits: c_int = 0,
+    accum_blue_bits: c_int = 0,
+    accum_alpha_bits: c_int = 0,
+    aux_buffers: c_int = 0,
+    samples: c_int = 0,
+    refresh_rate: c_int = consts.dont_care,
+    
+    stereo: bool = false,
+    srgb_capable: bool = false,
+    doublebuffer: bool = true,
+    
+    client_api: ClientApi = .opengl_api,
+    context_creation_api: ContextCreationApi = .native_context_api,
+    
+    context_version_major: c_int = 1,
+    context_version_minor: c_int = 0,
+    
+    context_robustness: ContextRobustness = .no_robustness,
+    context_release_behavior: ContextReleaseBehavior = .any_release_behavior,
+    
+    opengl_forward_compat: bool = false,
+    opengl_debug_context: bool = false,
+    
+    opengl_profile: OpenGlProfile = .opengl_any_profile,
+    
+    cocoa_retina_framebuffer: bool = true,
+    
+    cocoa_frame_name: [:0]const u8 = "",
+    
+    cocoa_graphics_switching: bool = false,
+    
+    x11_class_name: [:0]const u8 = "",
+    x11_instance_name: [:0]const u8 = "",
+    
+    pub const ClientApi = enum(c_int) {
+        opengl_api = consts.opengl_api,
+        opengl_es_api = consts.opengl_es_api,
+        no_api = consts.no_api,
+    };
+    
+    pub const ContextCreationApi = enum(c_int) {
+        native_context_api = consts.native_context_api,
+        egl_context_api = consts.egl_context_api,
+        osmesa_context_api = consts.osmesa_context_api,
+    };
+    
+    pub const ContextRobustness = enum(c_int) {
+        no_robustness = consts.no_robustness,
+        no_reset_notification = consts.no_reset_notification,
+        lose_context_on_reset = consts.lose_context_on_reset,
+    };
+    
+    pub const ContextReleaseBehavior = enum(c_int) {
+        any_release_behavior = consts.any_release_behavior,
+        release_behavior_flush = consts.release_behavior_flush,
+        release_behavior_none = consts.release_behavior_none,
+    };
+    
+    pub const OpenGlProfile = enum(c_int) {
+        opengl_any_profile = consts.opengl_any_profile,
+        opengl_compat_profile = consts.opengl_compat_profile,
+        opengl_core_profile = consts.opengl_core_profile,
+    };
+    
+    fn set(hints: Hints) !void {
+        inline for (comptime std.meta.fieldNames(Hints)) |field_name| {
+            const hint_tag = @enumToInt(@field(Hint, field_name));
+            const hint_value = @field(hints, field_name);
+            switch (@TypeOf(hint_value)) {
+                bool => c.glfwWindowHint(hint_tag, @boolToInt(hint_value)),
+                c_int => c.glfwWindowHint(hint_tag, hint_value),
+                
+                ClientApi,
+                ContextCreationApi,
+                ContextRobustness,
+                ContextReleaseBehavior,
+                OpenGlProfile,
+                => c.glfwWindowHint(hint_tag, @enumToInt(hint_value)),
+                
+                ([:0]const u8) => c.glfwWindowHintString(hint_tag, hint_value.ptr),
```

Comment:
Any particular reason for these parens? Personally I find it more readable without them

Reply:
My personal reason for it is that I don't like "mixing" whitespace "scopes" (the obligatory space between 'const' and 'u8', and the optional whitespace between 'u8' and '=>'), if that makes any sense. Though, I have no particular attachment to it either, just a sort of typing habit. I can go ahead and change it whenever I get on my laptop.

Reply:
Happy to have this as-is and let `zig fmt` formally sort it out later.